cubi-tools/update_workflow.py

```diff
@@ -5,6 +5,7 @@
 import subprocess as sp
 import argparse as argp
 import toml
+import requests
```

Comment:
you introduced an additional dependency and I don't see why that is necessary, you only use it to check if the repo exists, which can simply be done by trying to access it and assuming it's invalid if that fails.

Reply:
Has been removed